restapi/tenants.go

```diff
@@ -63,6 +63,38 @@ func registerTenantHandlers(api *operations.M3API) {
 		return admin_api.NewTenantInfoOK().WithPayload(resp)
 
 	})
+
+	// Detail Tenant
+	api.AdminAPIDeleteTenantHandler = admin_api.DeleteTenantHandlerFunc(func(params admin_api.DeleteTenantParams) middleware.Responder {
+		err := getDeleteTenantResponse(params)
+		if err != nil {
+			return admin_api.NewTenantInfoDefault(500).WithPayload(&models.Error{Code: 500, Message: swag.String(err.Error())})
+		}
+		return admin_api.NewTenantInfoOK()
+
+	})
+}
+
+// getDeleteTenantResponse gets the output of deleting a minio instance
+func getDeleteTenantResponse(params admin_api.DeleteTenantParams) error {
+	opClientClientSet, err := operatorClientset.NewForConfig(cluster.GetK8sConfig())
+	if err != nil {
```

Comment:
can you add logs for the errors here and on the call of `deleteTenantAction()`

Reply:
we only need to log once the same error, done